### PR TITLE
Final touches for agency and saas2

### PR DESCRIPTION
**Pull request by AMhaish**

### CODE REVIEW

File: bootstrapers/showcase/mappers/banners.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import PortfolioBannerSection from '../../../lib/containers/Portfolio/Banner';
-import SaaSBannerSection from '../../../lib/containers/Saas/BannerSection';
+import SaaS2BannerSection from '../../../lib/organisms/components/banners/banner3';
```
Comment (AMhaish) [inline bootstrapers/showcase/mappers/banners.js:2]: Please when you work on SaaS theme (SaaS Classic), let's fix the naming of components so the theme you worked on is SaaS2 so her components should be named like "SaaS2BannerSection"
And the new one is SaaS so her banner name would be "SaaSBannerSection"

File: lib/organisms/components/banners/banner3/index.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import BannerWrapper, {
-  DiscountLabel,
-  BannerObject,
-} from './style';
-import BannerObject1 from '../../../assets/image/saas/banner/bannerObject1.png';
-import BannerObject2 from '../../../assets/image/saas/banner/bannerObject2.png';
-import { Button, Container, Box, Heading,Text,Image } from '../../../../atoms';
```
Comment (AMhaish) [inline lib/organisms/components/banners/banner3/index.tsx:10]: Let's remove unneeded imports

File: lib/organisms/components/footers/footer3/index.tsx [hunk 3]
```diff
@@ -93,17 +95,6 @@ const Footer = ({
...
-Footer.propTypes = {
-  row: PropTypes.object,
-  col: PropTypes.object,
-  colOne: PropTypes.object,
-  colTwo: PropTypes.object,
-  titleStyle: PropTypes.object,
-  textStyle: PropTypes.object,
-  logoStyle: PropTypes.object,
-};
-
```
Comment (AMhaish) [inline lib/organisms/components/footers/footer3/index.tsx:106]: Let's remove propTypes while they are not used anymore, no need to repeat that for old themes, just for new ones whenever you see them

File: bootstrapers/showcase/mappers/banners.js (hunk 1 above)
Comment (RanimHelamiah) [reply bootstrapers/showcase/mappers/banners.js:2]: done

File: lib/organisms/components/banners/banner3/index.tsx (hunk 2 above)
Comment (RanimHelamiah) [reply lib/organisms/components/banners/banner3/index.tsx:10]: done

File: lib/organisms/components/footers/footer3/index.tsx (hunk 3 above)
Comment (RanimHelamiah) [reply lib/organisms/components/footers/footer3/index.tsx:106]: done

**Pull request merged**